#! /bin/sh -e

if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
elif [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch $pdir -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch $pdir -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
esac
exit 0

# DP: Clarify documentation about case-insensitive pathname expansion

--- bash/doc/bash.1~	2004-08-28 18:25:23.000000000 +0200
+++ bash/doc/bash.1	2004-08-28 19:13:24.000000000 +0200
@@ -2690,6 +2690,10 @@
 .B nocaseglob
 is enabled, the match is performed without regard to the case
 of alphabetic characters.
+Note that when using range expressions like
+[a-z] (see below), letters of the other case may be included,
+depending on the setting of
+.B LC_COLLATE.
 When a pattern is used for pathname expansion,
 the character
 .B ``.''
